test/built-ins/Map/groupBy/string.js

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2023 Ecma International.  All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-map.groupby
+description: Map.groupBy works for string items
+info: |
+  Map.groupBy ( items, callbackfn )
+  ...
+includes: [compareArray.js]
+features: [array-grouping, Map]
+---*/
+
+const string = 'abcd';
```

Comment:
let's please add a surrogate pair into the test strings. i typically use 💩 but pick your favorite :-)

Reply:
I chose four emojis. [961e633](https://github.com/tc39/test262/pull/4038/commits/961e63382699256ec7f072a9255c9bf6fc619ed9)